manager.py

```diff
@@ -92,19 +93,8 @@ def configure(self, context):
             "configuration", "start_stop_services", fallback=True)
         self._delete_missing = Settings.getboolean(
             "configuration", "delete_missing", fallback=True)
-
-        # fetch instance specific settings
-        self._api_key = Persistence().load(
-            "api_key",
-            default=Settings.get("configuration", "instance_api_key"))
-        self._instance_id = Persistence().load(
-            "instance_id",
-            default=Settings.get("configuration", "instance_id"))
-
-        # config autonomy specific settings
-        self._poll_interval = Settings.getint("configuration",
-                                              "config_poll_interval",
-                                              fallback=0)
+        self._poll_interval = Settings.getint(
+            "configuration", "config_poll_interval", fallback=0)
```

Comment:
The definitions in the __init__ method for these two members `_api_key` and `_instance_id` should be removed